test/unit/core.js

```diff
@@ -489,8 +493,56 @@ QUnit.test( "isFunction", function( assert ) {
 	callme( function() {
 		callme( function() {} );
 	} );
+
+	// Functions from other windows should be matched
+	Globals.register( "iframeDone" );
+	window.iframeDone = function( fn, detail ) {
+		window.iframeDone = undefined;
+		assert.ok( jQuery.isFunction( fn ), "cross-realm function" +
+			( detail ? " - " + detail : "" ) );
+		done();
+	};
+
+	try {
+		iframe = jQuery( "#qunit-fixture" )[ 0 ].appendChild( document.createElement( "iframe" ) );
+		doc = iframe.contentDocument || iframe.contentWindow.document;
+		doc.open();
+		doc.write( "<body onload='window.parent.iframeDone( function() {} );'>" );
+		doc.close();
+	} catch ( e ) {
+		window.iframeDone( function() {}, "iframes not supported" );
```

Comment:
The block was patterned off a [similar one for isPlainObject](https://github.com/jquery/jquery/blob/ac9e3016645078e1e42120822cfb2076151c8cbe/test/unit/core.js#L363), and is [quite old](https://github.com/jquery/jquery/commit/396dd2127330a7ed742d1e9092af54b668c46e85#diff-023537a1eb6e158db4d12a1fe86f1128R262) as it turns out. I'll remove the try/catch.